src/http/response.ts

```diff
@@ -1,28 +1,17 @@
 import * as Drash from "../../mod.ts";
 
-export interface IOptions {
-  default_content_type?: string;
-}
+export class DrashResponse {
 
-/**
- * This class represents a response that is generated by a resource -- to be
- * sent back to the client.
- */
-export class DrashResponse extends Response {
-  public headers = new Headers();
-  public status = 200;
-  #options: Drash.Interfaces.IResponseOptions = {};
+  public body: BodyInit | null | undefined;
+  public headers: Headers = new Headers();
+  public status: number = 200;
```

Comment:
no need for this as its already present on the `Response` right? plus, it means we can parse `NativeResponse#body`

Reply:
we need this or else we can't use `this.response.body` in the resource. if we set `Resource.response = new Response()`, we can't mutate the `this.response.body` because it's readonly